en/components/general/ignite-ui-licensing.md

```diff
@@ -119,6 +119,77 @@ If you are upgrading from trial to licensed package and you are not using the au
 So, if you've already adopted npm and you have an Ignite UI for Angular license, don't hesitate setting up the Infragistics private feed and boost your productivity, using the full potential of Ignite UI for Angular.
 
 
-### [TODO] How to set up CI provider
+## Access Token Usage
 
-**Describe** a way to integrate a Public CI process in a repository, which uses a licensed package (with access token) or a way to acquire a access token through infragistics.com user account, which later will be used to access the licensed package.
+You can acquire and use a access token through your [infragistics.com user account](https://account.infragistics.com/access-tokens), if you want to integrate a CI process in a repository and use it to access the Ignite UI for Angular licensed packages.
+
+Once generated the token can be used on Azure Pipelines build procedures, Travis CI or local configurations: 
+
+* Generate a token from https://account.infragistics.com/access-tokens
+
+<img class="responsive-img" style="-webkit-box-shadow: 8px 9px 9px 5px #ccc; -moz-box-shadow: 8px 9px 9px 5px #ccc; box-shadow: 8px 9px 9px 5px #ccc; width: calc(100% - 250px)" 
+  src="../../images/general/generate-token.jpg"
+  data-src="../../images/general/generate-token.jpg"
+  alt="New Token Generated"
+  title="Generate new token" />
+
+> Each token is with Base64 encoding.
+
+* Add the following into your [.npmrc](https://docs.npmjs.com/configuring-npm/npmrc.html) file
+
+```cmd
+@infragistics:registry=https://packages.infragistics.com/npm/js-licensed/
+//packages.infragistics.com/npm/js-licensed/:_auth={YOUR_ACCESS_TOKEN}
+//packages.infragistics.com/npm/js-licensed/:username:{YOUR_USERNAME}
+//packages.infragistics.com/npm/js-licensed/:always-auth=true
+```
+
+### Azure Pipelines Configuration
+Update the azure-pipelines.yml with the following steps:
+
+```cmd
+npm config set @infragistics:registry $(npmRegistry)
+```
+
+```cmd
+npm config set always-auth true --scope:@infragistics
+```
+
+```cmd
+npm config set _auth=$(token) --scope:@infragistics
```

Comment:
I'd advice against using those commands as a whole right now, since the scope parameter here does nothing (i.e. the set command doesn't accept such argument) and thus this ends up as a global setting which may or may not interfere with potential other credentials for npm registry/Azure artifacts/nexus etc. Can't be a 100% sure what, but https://stackoverflow.com/questions/49404870/how-to-set-auth-for-a-scoped-registry-in-npmrc seems to suggest those don't always work as expected and they don't get prefixed with the registry URI in the config

Reply:
Wouldn't it be better to mention this with a Warning Note and leave it as is?

Reply:
I've logged a separate [issue](https://github.com/IgniteUI/igniteui-angular-samples/issues/2060) for your comment. We are going to address it after further research on the subject.